### ISSUE TITLE
Content on Hover or Focus (WCAG 1.4.13a)

### ISSUE THREAD

**Issue by VivienDZB** [slidewiki/slidewiki-platform#1232]
**Request:** Additional content that appears and disappears in coordination with keyboard focus or pointer hover often leads to accessibility issues. Examples of such interactions can include custom tooltips, sub-menus and other nonmodal popups which display on hover and focus. The intent of this success criterion is to ensure that authors who cause additional content to appear and disappear in this manner must design the interaction in such a way that users can perceive the additional content and dismiss it without disrupting their page experience.

**Issues:**
1. https://slidewiki.org/ 
- The "Skip to main content" content that opens with the keyboard focus cannot be closed by pressing Escape. The content becomes invisible with a mouse hover. 
- This also affects the following pages:

2. https://slidewiki.org/presentation/81127-1/overview-of-accessibility/811271/437649-3/?language=en-/slide-719796-3 
3. a. https://slidewiki.org/deck/127411/_/slide/836159-2/836159-2:1/edit 
b. https://slidewiki.org/deck/127411/_/slide/836159-2/836159-2:1/markdownEdit 
4. https://slidewiki.org/search?keywords=accessibility&sort=score 
5. https://slidewiki.org/user/vivien.zschammer/recommendations 
6. https://slidewiki.org/addDeck 
7. https://slidewiki.org/contactus 
8. https://slidewiki.org/deck/127411/test/deck/127411 
9. https://slidewiki.org/accessibility 

10. a. https://slidewiki.org/user/vivien.zschammer/settings/profile
b. https://slidewiki.org/user/vivien.zschammer/stats 
- See page 1.
- The tooltips displayed in the coordinate system by hovers cannot be closed by clicking. When the pointer moves over the displayed content, it does not remain constant. 

Tooltips should close by clicking or pressing the escape key, but should remain constant with a mouse hover.

**Comment by aoelen** [comment]
I fixed the issue with the invisible skip link. W.r.t. to closing this area on escape. Is this something that is really required? We are using the following package: https://designsystem.gov.au/components/skip-link/ which does not support that. Also on this specific website, they implemented the skip links, but they don't disappear on escape.

**Comment by VivienDZB** [comment]
It is recommended by WCAG. The website says that there are two possible ways to satisfy the condition:

1. Position the additional content so that it does not obscure any other content including the trigger, with the exception of white space and purely decorative content, such as a background graphic which provides no information.
2. Provide a mechanism to easily dismiss the additional content, such as by pressing Escape.

It closes by pressing Tab. I'm not really sure if this is enough. I need to ask my colleague again, who is still on vacation until next week.
